tests/conftest.py

```diff
@@ -277,6 +277,14 @@ def pytest_configure(config):
             EntropyGenerator.minimum_entropy, EntropyGenerator.max_minutes
         ),
     )
+    config.addinivalue_line(
+        "markers",
+        "skip_initial_gh_actions_failure(skip=<boolean or callable, reason=None): Skip known test failures "
+        "under the new GH Actions setup if the environment variable SKIP_INITIAL_GH_ACTIONS_FAILURES "
+        "is equal to '1' and the 'skip' keyword argument is either `True` or it's a callable that "
+        "when called returns `True`. If `skip` is a callable, it should accept a single argument "
+        "'grains', which is the grains dictionary.",
+    )
```

Comment:
This is for your initial phase, yeah?  Ideally we get all these fixed in the future?

Reply:
Yeah, this marker is setup to set it to false and see what needs fixing.